themes/demo/layouts/default.htm

```diff
@@ -19,7 +24,7 @@
 
         <!-- Header -->
         <header id="layout-header">
-            {% partial 'site/header' %}
+            {% partial 'site/header' backend_url=backend_url %}
```

Comment:
You can just implement this in the partial itself, see https://octobercms.com/docs/cms/partials#partial-life-cycle

Reply:
Hmmm. I tried this before implement it in the layout, but I surely made a crap because it just displayed the php code block as raw text.

Will modify my crap.